sinclair-vod-feed-spec.md

```diff
@@ -176,14 +176,14 @@ This object represents a series, such as a season of a TV Show or a mini-series.
 | shortDescription | string | Required | A description that does not exceed 200 characters.
 | longDescription | string | Optional | A longer description exceeding 200.
 | tags | [Tag Object](#tag) | Optional | One tag object.
-| groups | [Group Object](#group) | Optional | An array of groups (episodes grouped together).
+| groups | [Group Object](#group) | Required* | An array of groups (episodes grouped together).
```

Comment:
Why is this marked as a Required property if we can't have a mixture of seasons, episodes, and groups?

Reply:
Seems like they should all be marked as optional

Reply:
I have added addition information for required fields. Series should have either of one

> :information_source: * Must have either `seasons` or `episodes` or `groups`. It cannot be mixed of all